### DISCUSSION TITLE
ModuleNotFoundError when getting started, probably PEBKAC

### DISCUSSION THREAD

**Discussion by 1larity** [coqui-ai/TTS#2459] [General Q&A]
Hi there, I'm pretty new to Python, I've installed and used several new libraries today But Coqui TTS defeats me
I've followed the instructions at https://tts.readthedocs.io/en/latest/inference.html, but I can't get the first synthesis example to run, which fails on the Import with _ModuleNotFoundError: No module named 'TTS.api'; 'TTS' is not a package_ 
I've successfully run the web server, and enumerated the model list from the CLI command _tts --list_models_, so it looks like (most) things are installed correctly. 
So far I have tried uninstalling TTS and re installing, and pip install --force-reinstall TTS.
I'm not sure if it's relevant but I have previously installed pyttsx3 maybe there is a clash? I am also using Conda.

**Comment by p0p4k** [comment]
Hi, did you activate your conda environment first?

**Reply by p0p4k** [reply]
the env where TTS is installed?

**Reply by 1larity** [reply]
Yeah I have it in my bashrc, and Vscode terminal init. I've kind of worked around the problem by passing shell commands out of my program to the CLI version, and oh boy, this project is amazing!
